lib/galaxy/datatypes/converters/parquet_to_csv_converter.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+Input: parquet
+Output: csv
+"""
+import os
+import sys
+from pyarrow import csv
+from pyarrow import parquet
```

Comment:
```suggestion
try:
    from pyarrow import csv
    from pyarrow import parquet
except ImportError:
    pyarrow = None
```

Reply:
Something like this @mvdbeek ?

Reply:
Yep, and then inside main function raise an appropriate exception

Reply:
oh, and I think it needs to be `csv = parquet = None`